backend/app/__init__.py

```diff
@@ -21,13 +23,22 @@ def create_app():
     api = Api(app)
     api.add_resource(ContentListResource, '/contents')
     api.add_resource(ContentResource, '/contents/<string:id>')
+    api.add_resource(UserListResource, '/users')
+    api.add_resource(UserResource, '/users/<string:user_id>')
     api.add_resource(UserRegisterResource, '/register')
     api.add_resource(UserLoginResource, '/login')
     app.config.from_object('config.Config')
 
     db.init_app(app)
     limiter.init_app(app)
 
+    # @app.errorhandler(Exception)
+    # def handle_exception(e):
+    #     """Handle uncaught exceptions"""
```

Comment:
Why is this code still here?

- [ ] Cleanup dead code.

Reply:
Actually it is important to handle all other exceptions not handled in the code. I have uncommented it now. I commented it out in order to be able to see all the error messages for debugging purposes